ichnaea/locate/tests/test_provider.py

```diff
@@ -53,9 +54,11 @@ class TestProvider(Provider):
     def setUpClass(cls):
         DBTestCase.setUpClass()
         GeoIPIsolation.setup_geoip(raven_client=cls.raven_client)
+        RedisIsolation.setup_redis()
 
     @classmethod
     def tearDownClass(cls):
+        RedisIsolation.teardown_redis()
         GeoIPIsolation.teardown_geoip()
         DBTestCase.tearDownClass()
 
```

Comment:
Did you solve the redis isolation problem? If not, you might want to add this bit, so redis gets cleaned after each test method:

```
def tearDown(self):
    self.cleanup_redis()
    self.teardown_session()
```
That's what AppTestCase does, which is almost the same combination of base classes, except that it also sets up a wsgi app.